src/components/Resources/ResourceCard.tsx

```diff
@@ -138,23 +139,34 @@ export default function ResourceCard({ resource, onUpdate }: IResourceCard) {
             />
           ))}
         </div>
-        <ResourceVoteControls
-          resource={resource}
-          onLike={() => {
-            if (user) {
-              likeMutation.mutate()
-            } else {
-              showUnauthorizedToast()
-            }
-          }}
-          onDislike={() => {
-            if (user) {
-              dislikeMutation.mutate()
-            } else {
-              showUnauthorizedToast()
-            }
-          }}
-        />
+        <div className="d-flex flex-row flex-wrap gap-2 align-items-center text-dark">
+          <ResourceVoteControls
+            resource={resource}
+            onLike={() => {
+              if (user) {
+                likeMutation.mutate()
+              } else {
+                showUnauthorizedToast()
+              }
+            }}
+            onDislike={() => {
+              if (user) {
+                dislikeMutation.mutate()
+              } else {
+                showUnauthorizedToast()
+              }
+            }}
+          />
+          <Link
+            to={`/resources/${resource.id}`}
+            className="text-decoration-none"
+          >
+            <div className={'d-flex flex-row gap-2'}>
```

Comment:
To match the voting control styles and keep everything consistant: 
```suggestion
            <div className={'d-flex flex-row gap-1 text-dark'}>
```